.circleci/config.yml

```diff
@@ -347,25 +347,6 @@ jobs:
             - purchases/test_artifacts/integrationTest-app.apk
             - purchases/test_artifacts/integrationTest-test.apk
 
-  purchases-load-shedder-integration-tests-build:
-    <<: *android-executor
-    steps:
-      - checkout
-      - install-sdkman
-      - revenuecat/install-gem-unix-dependencies:
-          cache-version: v1
-      - android/restore-build-cache
-      - run:
-          name: Create purchases integration tests apks
-          command: |
-            bundle exec fastlane android build_load_shedder_purchases_integration_tests
-      - android/save-build-cache
-      - persist_to_workspace:
-          root: .
-          paths:
-            - purchases/test_artifacts/loadShedderIntegrationTest-app.apk
-            - purchases/test_artifacts/loadShedderIntegrationTest-test.apk
```

Comment:
Now this is moved inside the `run-firebase-tests-purchases-load-shedder-integration-test` job